lib/casbin/persist/filtered_adapter.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'casbin/persist/adapter'
+
+module Casbin
+  module Persist
+    # FilteredAdapter is the interface for Casbin adapters supporting filtered policies.
+    class FilteredAdapter < Casbin::Persist::Adapter
+      # IsFiltered returns true if the loaded policy has been filtered
+      # Marks if the loaded policy is filtered or not
+      def is_filtered; end
```

Comment:
Maybe use Ruby-specific semantics for boolean methods? Like `filtered?`

Reply:
I assumed the `filtered?` name for this method [here](https://github.com/evrone/casbin-ruby/pull/20)